lib/phoenix/code_reloader.ex

```diff
@@ -32,6 +32,15 @@ defmodule Phoenix.CodeReloader do
+    monospace_font: "menlo, consolas, monospace"
```

Comment:
I don't quite like how the logo base64 is repeated here (also seen in `Phoenix.Endpoint`). Any ideas?